signature-specification.md

```diff
@@ -1,243 +1,118 @@
-The document has the following sections:
+This document provides the following details for Notary v2 signatures:
 
-- **[Storage](#storage)**: Describes how signatures are stored in OCI registry.
-- **[Signature Envelope](#signature-envelope)**: Describes how signatures are created.
+- **[Storage](#storage)**: Describes how signatures are stored and retrieved from an OCI registry.
+- **[Signature Envelope](#signature-envelope)**: Describes the structure of a Notary v2 signature.
 
 ## Storage
 
 This section describes how Notary v2 signatures are stored in the OCI Distribution conformant registry.
-Notary v2 uses [ORAS artifact manifest](https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest.md) to store the signature in the repository.
+Notary v2 uses [ORAS artifact manifest][oras-artifact-manifest] to store the signature in the repository.
 The media type of the signature manifest is `application/vnd.cncf.oras.artifact.manifest.v1+json`.
 The signature manifest has an artifact type that specifies it's a Notary V2 signature, a reference to the manifest of the artifact being signed, a blob referencing the signature, and a collection of annotations.
 
 ![Signature storage inside registry](media/signature-specification.svg)
 
 - **`artifactType`** (*string*): This REQUIRED property references the Notary version of the signature: `application/vnd.cncf.notary.v2.signature`.
-- **`blobs`** (*array of objects*): This REQUIRED property contains collection of only one [artifact descriptor](https://github.com/oras-project/artifacts-spec/blob/main/descriptor.md) referencing signature envelope.
-  - **`mediaType`** (*string*): This REQUIRED property contains media type of signature envelope blob. The supported value is `application/jose+json`
+- **`blobs`** (*array of objects*): This REQUIRED property contains collection of only one [artifact descriptor][artifact-descriptor] referencing signature envelope.
+  - **`mediaType`** (*string*): This REQUIRED property contains media type of signature envelope blob. Following values are supported
+    - `application/jose+json`
 - **`subject`** (*descriptor*): A REQUIRED artifact descriptor referencing the signed manifest, including, but not limited to image manifest, image index, oras-artifact manifest.
 - **`annotations`** (*string-string map*): This REQUIRED property contains metadata for the artifact manifest.
   It is being used to store information about the signature.
   Keys using the `io.cncf.notary` namespace are reserved for use in Notary and MUST NOT be used by other specifications.
-  - **`io.cncf.notary.x509certs.fingerprint.sha256`**: A REQUIRED annotation whose value contains the list of SHA-256 fingerprint of signing certificate and certificate chain used for signature generation.
+  - **`io.cncf.notary.x509.fingerprint.sha256`**: A REQUIRED annotation whose value contains the list of SHA-256 fingerprint of signing certificate and certificate chain used for signature generation.
```

Comment:
How about `io.cncf.notary.x5t#S256` in [RFC7515 4.1.8](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.8)?

`x5t#S256` looks more common and shorter and it also defines the encoding (i.e. a base64url-encoded SHA-256 thumbprint). However, it seems only defines the cert for the signing key and is not a list.

Reply:
Renamed to `io.cncf.notary.x509chain.thumbprint#S256`, let me know if you have another suggestion.